test.md

```diff
@@ -0,0 +1 @@
+"This file should be ignored by git".
```

Comment:
- [ ] Great job 👌at adding the ``` test.md ``` file at the gitignore, but there are some issues 😃you need to work I see that you already put the ``` test.md``` file at the gitignore, but it is still existing at the repo on GitHub😊, so kindly remove the ``` test.md``` file from the remote repo as it is from the [project requirements](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md)💯